oca_dependencies.txt

```diff
@@ -0,0 +1 @@
+server-ux
```

Comment:
this file is useles

Reply:
Hum, from what has it been replaced?

Reply:
Is useless on:

1. old repo has "PIP" install mode instead of "OCA"
2. all new repos using GH actions